src/MerkleMountainRange.sol

```diff
@@ -34,7 +34,7 @@ library MerkleMountainRange {
     /// @param leaves a list of mmr leaves to prove
     /// @return boolean if the calculated root matches the provides root node
     function VerifyProof(bytes32 root, bytes32[] memory proof, MmrLeaf[] memory leaves, uint256 mmrSize)
-        public
+        internal
```

Comment:
For our project needs, we can't embed this contract. It needs to be deployed seperately as it is too large

Reply:
> For our project needs, we can't embed this contract. It needs to be deployed seperately as it is too large

i think if use public  , the type library need change to type contract